inspector/an-inspector/ForgeModule/src/io/trigger/forge/android/modules/share_sheet/API.java

```diff
@@ -0,0 +1,38 @@
+package io.trigger.forge.android.modules.share_sheet;
+
+import io.trigger.forge.android.core.ForgeApp;
+import io.trigger.forge.android.core.ForgeParam;
+import io.trigger.forge.android.core.ForgeTask;
+import android.content.Intent;
+
+public class API {
+
+	private static final String FOOTER = "\n\nvia Fetchnotes (http://fetchnotes.com/)";
+
+	/**
+	 * @param task Forge.IO task object, used to return data and trigger
+	 * @param text Text content of the note
+	 * @param url First URL in the note, if any
+	 * @param archived 'true' if the note has been archived, or 'false'
+	 * @param publicLink URL to view the note on Fetchnotes' web app
+	 */
+	public static void show(final ForgeTask task,
+			@ForgeParam("text") final String text,
+			//@ForgeParam("url") final String url,
+			@ForgeParam("archived") final String archived,
```

Comment:
It's not currently used, and an error is thrown if `url` isn't provided (which it often isn't).

There's ways to handle it as an optional parameter, but i figured the easiest thing to do was just comment it out.